src/components/MultiHopTrade/components/LimitOrderV2/LimitOrderConfirm.tsx

```diff
@@ -321,6 +320,13 @@ export const LimitOrderConfirm = () => {
 
         setLimitOrderTxComplete(quoteId)
 
+        if (result.data) {
```

Comment:
q: do we want to early return here instead? Not sure about tags invalidation, but mixpanel success event if the order placing failed seems off

Reply:
Correct, the previous condition was wrong, it's fine now in [cdf0b44](https://github.com/shapeshift/web/pull/9626/commits/cdf0b44a2fa6ccc6a07536159b35c86d1f1a1a26)